modules/nf-core/lofreq/callparallel/main.nf

```diff
@@ -23,15 +23,25 @@ process LOFREQ_CALLPARALLEL {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def options_intervals = intervals ? "-l ${intervals}" : ""
+
+    def tumor_cram =  tumor.Extension == "cram" ? true : false
+    def tumor_bam = tumor.Extension == "bam" ? true : false
+    def tumor_out = tumor_cram ? tumor.BaseName + ".bam" : "${tumor}"
+
+    def samtools_cram_convert = ''
```

Comment:
If you are converting the files, please remove them at the end, don't want 50Gb bam files lying around...